pom.xml

```diff
@@ -84,6 +84,11 @@
             <version>3.0.3</version>
             <type>jar</type>
         </dependency>
+        <dependency>
+            <groupId>com.alibaba</groupId>
+            <artifactId>fastjson</artifactId>
+            <version>1.2.47</version>
```

Comment:
i highly recommend using http://plugins.jenkins.io/jackson2-api unless there is a specific need to use a custom JSON library

Reply:
OK, I will change it.